examples/decorator/noxfile.py

```diff
@@ -3,7 +3,5 @@
 
 @nox.session
 def python(session):
-    session.install("-rrequirements-dev.txt")
-    session.install("maturin")
-    session.run_always("maturin", "develop")
+    session.install(".[dev]", "--config-settings=build-args=--profile=dev")
```

Comment:
This thrice nested argument passing looks quite unreadable. While putting everything into `pyproject.toml` is nice, is there no way around this incantation?

Reply:
@messense any suggestions? I think we deprecated `MATURIN_PEP517_ARGS` so unless pip has an environment variable for this I think command line is the only way.

Reply:
`--config-settings` is the recommended way to pass arguments, but I'm thinking about un-deprecate `MATURIN_PEP517_ARGS` because Poetry doesn't seem to support `--config-settings`.

Reply:
In that case @adamreichold would you prefer I switch to the env var here?

@messense see also https://github.com/PyO3/setuptools-rust/issues/327#issuecomment-1525560005 which was another reason to want an environment variable.

Reply:
> In that case @adamreichold would you prefer I switch to the env var here?

Yes, I think it would be preferable if it removes at least one level of `=` nesting.

Reply:
Done 👍